test-it/ViaTests/JSReferenceCloseArray.via

```diff
@@ -0,0 +1,28 @@
+define(ArrayOfReferencesVI dv(VirtualInstrument (
+    Locals: c(
+        e(a(.JavaScriptDynamicRefNum *) arrayOfReferences)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) closeReferenceError)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) getObjectReferenceError)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) invalidReferenceError)
+
+        e(.JavaScriptDynamicRefNum objectReference1)
+        e(.JavaScriptDynamicRefNum objectReference2)
+        e(dv(.Boolean false) isNotAValidRefnum1)
+        e(dv(.Boolean false) isNotAValidRefnum2)
+        e(.Occurrence occurrence)
+    )
+    clump (
+        JavaScriptInvoke(occurrence false getObjectReferenceError 'Date.now' objectReference1)
+        JavaScriptInvoke(occurrence false getObjectReferenceError 'Date.now' objectReference2)
```

Comment:
before closing can you assert that the refnums are valid?

Reply:
Done.